### PR TITLE
Making round robin advance distance configurable

### PR DESCRIPTION
**Pull request by jamesjennens**

### CODE REVIEW

**Comment by jamesjennens** [comment]
I was thinking we just replicate the current setup in that we currently
have the ability to set the number of episodes so there's no alignment
between these two things now. I don't see any issues with that (but
perhaps overlooking something). However, if we're running on two different
tracks at the same time I can see a use for having separate config but
that's probably a larger issue shared with many other params.

**Comment by larsll** [comment]
Fair point @jamesjennens. At least possible to do increase of 10%, and then multiple workers getting 10 tasks each. (As 20 each is a notch much when in advanced training on long tracks...)

Some other advanced options (for later) is to have multiple workers with different start position Would be interesting if you have very many workers each doing e.g. 5 episodes.

**Comment by jamesjennens** [comment]
found a bug. will resubmit

**Comment by jamesjennens** [comment]
looks like i don't need to resubmit. the PR picks up the latest version of my branch?

File: bundle/sagemaker_rl_agent/lib/python3.5/site-packages/markov/rollout_worker.py [hunk 1]
```diff
@@ -286,6 +286,10 @@ def main():
                         help='(float) collision penalty second',
                         type=float,
                         default=float(rospy.get_param("COLLISION_PENALTY", 0.0)))
+    parser.add_argument('--round_robin',
+                        help='(float) round robin distance 0-1',
+                        type=float,
+                        default=float(rospy.get_param("ROUND_ROBIN", 0.05)))
```
Comment (larsll) [inline bundle/sagemaker_rl_agent/lib/python3.5/site-packages/markov/rollout_worker.py:292]: Can we call it ROUND_ROBIN_ADVANCE_DIST?

**Comment by jamesjennens** [comment]
sure. i'll make the change

**Pull request merged**